service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/JudgeAndLegalAdvisor.java

```diff
@@ -24,8 +26,8 @@
     @Builder(toBuilder = true)
     @SuppressWarnings("java:S107")
     private JudgeAndLegalAdvisor(JudgeOrMagistrateTitle judgeTitle, String otherTitle, String judgeLastName,
-                                String judgeFullName, String legalAdvisorName, String allocatedJudgeLabel,
-        String useAllocatedJudge, String judgeEmailAddress) {
+                                 String judgeFullName, String legalAdvisorName, String allocatedJudgeLabel,
+                                 String useAllocatedJudge, String judgeEmailAddress) {
```

Comment:
maybe look into removing in the future, I don't think it is needed, looks like it is just overwriting what lombok would generate